CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
 * [#5597](https://github.com/bbatsov/rubocop/pull/5597): Add new `Rails/HttpStatus` cop. ([@anthony-robin][])
 * [#5643](https://github.com/bbatsov/rubocop/pull/5643): Add new `Style/UnpackFirst` cop. ([@bdewater][])
+* [#5753](https://github.com/bbatsov/rubocop/pull/5753): Add new `Performance/UnneededSort` cop. ([@parkerfinch][])
```

Comment:
I didn't expand the preview on GH until now - sorry to be a bother - noticed this is under the '0.54.0' header instead of 'master' probably due to the rebase.

Reply:
Whew, good catch! Fixed.